src/latexify/latexify_visitor.py

```diff
@@ -171,26 +171,65 @@ def _decorated_lstr_and_arg(node, callee_str, lstr):
             lstr, arg_str = _decorated_lstr_and_arg(node, callee_str, lstr)
             return lstr + arg_str + rstr
 
-    def visit_Attribute(self, node, action):  # pylint: disable=invalid-name
+    def visit_Attribute(self, node, action):
         vstr = self.visit(node.value)
         astr = str(node.attr)
         return vstr + "." + astr
 
-    def visit_Name(self, node, action):  # pylint: disable=invalid-name
+    def visit_Name(self, node, action):
         if self._reduce_assignments and node.id in self.assign_var.keys():
             return self.assign_var[node.id]
 
         return self._math_symbol_converter.convert(str(node.id))
 
-    def visit_Constant(self, node, action):  # pylint: disable=invalid-name
-        # for python >= 3.8
-        return str(node.n)
+    def convert_constant(self, value: Any) -> str:
+        """Helper to convert constant values to LaTeX.
 
-    def visit_Num(self, node, action):  # pylint: disable=invalid-name
-        # for python < 3.8
-        return str(node.n)
+        Args:
+            value: A constant value.
+
+        Returns:
+            The LaTeX representation of `value`.
+        """
+        if value is None or isinstance(value, bool):
+            return r"\mathrm{" + str(value) + "}"
+        if isinstance(value, (int, float, complex)):
```

Comment:
Maybe some scientists prefer "i" to "j" ¯\\\_(ツ)\_/¯

> In contexts in which use of the letter i is ambiguous or problematic, the letter j or the Greek [ι](https://en.wikipedia.org/wiki/Iota) is sometimes used instead.[[a]](https://en.wikipedia.org/wiki/Imaginary_unit#cite_note-3) For example, in [electrical engineering](https://en.wikipedia.org/wiki/Electrical_engineering) and [control systems engineering](https://en.wikipedia.org/wiki/Control_systems_engineering), the imaginary unit is normally denoted by j instead of i, because i is commonly used to denote [electric current](https://en.wikipedia.org/wiki/Electric_current).

https://en.wikipedia.org/wiki/Imaginary_unit

Reply:
Haha, it would be better to provide some flag to switch the appearance of this value.